src/Worker/Grpc/GrpcDurableTaskWorker.cs

```diff
@@ -52,13 +50,13 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
     AsyncDisposable BuildChannel(out Channel channel)
     {
-        if (this.grpcOptions.Channel is Channel c)
+        if (this.options.Channel is Channel c)
         {
             channel = c;
             return default;
         }
 
-        string address = string.IsNullOrEmpty(this.grpcOptions.Address) ? "127.0.0.1:4001" : this.grpcOptions.Address!;
+        string address = string.IsNullOrEmpty(this.options.Address) ? "127.0.0.1:4001" : this.options.Address!;
```

Comment:
Same as with my other comment, wondering if you'd be willing to swap `127.0.0.1` for `localhost` here too...

Reply:
Done